main.go

```diff
@@ -121,14 +124,16 @@ func main() {
 		IdentityPubkey: getInfo.IdentityPubkey,
 	}
 
+	onePer10sMiddleware := createRateLimitMiddleware(10, 1)
 	// Public endpoints for account creation and authentication
 	e.POST("/auth", controllers.NewAuthController(svc).Auth)
-	e.POST("/create", controllers.NewCreateUserController(svc).CreateUser)
+	e.POST("/create", controllers.NewCreateUserController(svc).CreateUser, onePer10sMiddleware)
 
 	// Secured endpoints which require a Authorization token (JWT)
-	secured := e.Group("", tokens.Middleware(c.JWTSecret))
+	secured := e.Group("", tokens.Middleware(c.JWTSecret), middleware.RateLimiter(middleware.NewRateLimiterMemoryStore(1)))
```

Comment:
just put one each second for all secured, it can easily be changed with your suggestion